packages/common/styles/_card.scss

```diff
@@ -0,0 +1,13 @@
+.card {
+  border-radius: 20px;
+  border: 0;
+  box-shadow: 0px 6px 15px rgba(0, 0, 0, 0.25);
```

Comment:
Try to always stick to `rem` units.
At least we create an standard in our code in regards CSS units